README.rst

```diff
@@ -210,38 +210,8 @@ Please see our `contributing guide <https://github.com/mne-tools/mne-bids/blob/m
 Cite
 ----
 
-If you use ``mne-bids`` in your work, please cite one of the following papers,
-depending on which modality you used:
-
-`MEG <http://doi.org/10.1038/sdata.2018.110>`_
-##############################################
-
-.. code-block:: Text
-
-   Niso, G., Gorgolewski, K. J., Bock, E., Brooks, T. L., Flandin, G., Gramfort, A.,
-   Henson, R. N., Jas, M., Litvak, V., Moreau, J., Oostenveld, R., Schoffelen, J.,
-   Tadel, F., Wexler, J., Baillet, S. (2018). MEG-BIDS, the brain imaging data
-   structure extended to magnetoencephalography. Scientific Data, 5, 180110.
-   http://doi.org/10.1038/sdata.2018.110
-
-
-`EEG <https://doi.org/10.1038/s41597-019-0104-8>`_
-##################################################
-
-.. code-block:: Text
-
-   Pernet, C. R., Appelhoff, S., Gorgolewski, K. J., Flandin, G.,
-   Phillips, C., Delorme, A., Oostenveld, R. (2019). EEG-BIDS, an extension
-   to the brain imaging data structure for electroencephalography. Scientific
-   Data, 6, 103. https://doi.org/10.1038/s41597-019-0104-8
-
-
-`iEEG <https://doi.org/10.1038/s41597-019-0105-7>`_
-###################################################
+If you use ``mne-bids`` in your work, please cite the following paper:
 
 .. code-block:: Text
 
-   Holdgraf, C., Appelhoff, S., Bickel, S., Bouchard, K., D'Ambrosio, S.,
-   David, O., … Hermes, D. (2019). iEEG-BIDS, extending the Brain Imaging Data
-   Structure specification to human intracranial electrophysiology. Scientific
-   Data, 6, 102. https://doi.org/10.1038/s41597-019-0105-7
+    Appelhoff, S., Sanderson, M., Brooks, T., Vliet, M., Quentin, R., Holdgraf, C., Chaumon, M., Mikulan, E., Tavabi, K., Höchenberger, R., Welke, D., Brunner, C., Rockhill, A., Larson, E., Gramfort, A. and Jas, M. (2019). MNE-BIDS: Organizing electrophysiological data into the BIDS format and facilitating their analysis. Journal of Open Source Software 4: (1896).
```

Comment:
@sappelhoff @agramfort I updated this. Okay for you?

Reply:
I would ask to cite this one first and then "one of the reference papers".  So have both citations requested.